handsontable/src/pluginHooks.js

```diff
@@ -1365,6 +1365,27 @@ const REGISTERED_HOOKS = [
    */
   'afterPaste',
 
+  /**
+   * Fired by {@link ManualColumnFreeze} plugin before freezing a column.
+   *
+   * @event Hooks#beforeColumnFreeze
+   * @since 12.1.0
+   * @param {number} column Visual column index of frozen column.
+   * @param {boolean} freezePerformed Indicates if order of columns will be changed by freeze.
```

Comment:
```suggestion
   * @param {boolean} freezePerformed If freezing the column is going to affect the order of columns, returns `true`. Otherwise, returns `false`.
```

Reply:
@wszymanski - right, I updated the suggestion.